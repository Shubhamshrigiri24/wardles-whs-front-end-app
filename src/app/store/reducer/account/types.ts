import { TActionStatus } from "app/store/types";
import { ACCOUNT_ACTIONS } from "app/store/reducer/account/constants";

export type TAccount = {
  dob: string;
  email: string;
  firstName: string;
  gender: string;
  lastName: string;
  phone: string;
  updateTime: string;
  userId: string;
  addressline1: string;
  addressline2: string;
  agreeMarketing: boolean;
  city: string;
  gpDetails: string;
  postcode: string;
};

export type State = {
  data: TAccount | null;
  status: TActionStatus;
};

export type SetAccountDataActionType = {
  type: typeof ACCOUNT_ACTIONS.SET_ACCOUNT;
  payload: TAccount;
};

export type SetAccountStatusActionType = {
  type: typeof ACCOUNT_ACTIONS.SET_ACCOUNT_STATUS;
  payload: TActionStatus;
};

export type AccountActionTypes =
  | SetAccountDataActionType
  | SetAccountStatusActionType;
