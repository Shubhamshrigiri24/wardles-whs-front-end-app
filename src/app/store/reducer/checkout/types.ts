import { RSAA } from "redux-api-middleware";
import { PostcodeLookupAddress } from "../shared";

export interface State {
  customerDetails: CustomerDetails;
  confirmedPayment: boolean;
  braintreeToken: string;
  paymentErrorMessage: string;
  postCodeLookUpAddresses: PostcodeLookupAddress[];
  deliveryDetails: DeliveryDetails;
}

export interface CustomerDetails {
  title?: string;
  firstName: string;
  lastName: string;
  line1: string;
  line2: string;
  city: string;
  postcode: string;
  email: string;
  phone: string;
  surgeryName?: string;
  surgeryAddress?: string;
  allowMarketing?: boolean;
}

export interface AddCustomerDetailsAction {
  type: string;
  payload: {
    customerDetails: CustomerDetails;
  };
}

export interface DeliveryDetails {
  title?: string;
  firstName: string;
  lastName: string;
  line1: string;
  line2: string;
  city: string;
  postcode: string;
}

export interface AddDeliveryDetailsAction {
  type: string;
  payload: {
    deliveryDetails: DeliveryDetails;
  };
}

export interface FetchBraintreeTokenAction {
  type: string;
  [RSAA]: {
    endpoint: string;
    method: string;
    types: [string, string, string];
  };
}

export interface FetchBraintreeTokenResponseAction {
  type: string;
  payload: {
    token: string;
  };
}

export interface PaymentFailureAction {
  type: string;
  payload: {
    response: {
      message: string | null;
    } | null;
  };
}

export interface FetchAddressFromPostcodeActionResponse {
  type: string;
  payload: {
    addresses: PostcodeLookupAddress[];
  };
}
