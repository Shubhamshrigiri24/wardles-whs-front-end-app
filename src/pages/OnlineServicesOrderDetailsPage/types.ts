import { SUBSCRIPTION_STATUSES } from "app/store/reducer/subscriptions/constants";
import { Discount } from "app/store/reducer/order/types";

export type OrderDetailProductInfo = {
  id: string;
  productName: string;
  activeIngredient: string;
  imageURI: string;
  worksIn: string;
  duration: string;
  isBestValue: boolean;
  canBuy: boolean;
  description: string;
  moreInfo: {
    title: string;
    content: string;
  }[];
  subscriptionMoreInfo?: string;
  levels: {
    levelLabel: string;
    columnLabelTemplate: string;
    pricePerUnitTemplate?: string;
    shouldDeriveBestValue?: boolean;
    stealPriceFromSelectedChild?: boolean;
    renderOrder: number;
  }[];
  priceTemplate: string;
  buyLinkURITemplate: string;
  startConsultationLinkURI: string;
};

export type OrderPricingType = Discount & {
  delivery?: { code: "PREMIUM" | "FREE"; value: number };
};

export type OrderDetailsProduct = {
  name: string;
  sku: string;
  label: string;
  itemsPerPack: number;
  price: number;
  pricePerUnit: number;
  isBestValue: boolean;
  meta: Record<string, any>;
  productInfo: OrderDetailProductInfo;
};

export type Subscription = {
  startDate: string;
  renewalDate: string;
  nextShipment: string;
  status: typeof SUBSCRIPTION_STATUSES;
};

export type Payment = {
  cardNumber: string;
  hasError: boolean;
};

export type Delivery = {
  firstName: string;
  lastName: string;
  postcode: string;
  addressline1: string;
  addressline2: string;
  city: string;
  email: string;
  phone: string;
  product: any;
  pricing: OrderPricingType;
  shouldShowDeliveryInfo?: boolean;
};

export enum OrderType {
  SUBSCRIPTION = "subscription",
  INDIVIDUAL_ORDER = "order",
}

export enum ConfirmationState {
  INITIAL = "initial",
  CONFIRMED = "confirmed",
}
