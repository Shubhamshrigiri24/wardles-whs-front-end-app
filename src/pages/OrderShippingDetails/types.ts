import { OrderCustomerDetails } from "app/store/reducer/order/types";

export type AddressValues = {
  line1: string;
  line2: string;
  city: string;
  postcode: string;
};

export type FormValues = OrderCustomerDetails & {
  agreeTerms: boolean;
};

export type PostcodeAddress = {
  city: string;
  country: string;
  postcode: string;
  line1: string;
  line2: string;
};

export type Scenarios = {
  isAuthenticatedAndHasAccount: boolean;
  isAuthenticatedAndRequestsAccount: boolean;
  isNotAuthenticatedAndRequestsAccount: boolean;
  isAuthenticatedAndHasRequestedAccount: boolean;
  isNotAuthenticatedAndDoesNotRequestAccount: boolean;
};

export enum FormStep {
  ACCOUNT_DETAILS = "ACCOUTN_DETAILS",
  DELIVERY_DETAILS = "DELIVERY_DETAILS",
}
