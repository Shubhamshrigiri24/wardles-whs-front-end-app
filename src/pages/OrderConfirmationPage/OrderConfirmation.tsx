import React, { useEffect } from "react";
import {
  Container,
  Card,
  makeStyles,
  Theme,
  Typography,
  Divider,
} from "@material-ui/core";
import USPBanner from "../../components/USPBanner";
import CheckoutHeader from "../../components/CheckoutHeader/CheckoutHeader";
import {
  Product,
  OnlineService,
  Ed,
  Pack,
  OnlineQuestion,
} from "app/store/reducer/online/types";
import { analytics } from "@welldigital/ui-common/Analytics";
import OnlineFooter from "../../components/OnlineFooter/OnlineFooter";
import ImportantInfo from "./ImportantInfo/ImportantInfo";

const useStyles = makeStyles((theme: Theme) => ({
  pageMargin: {
    marginTop: theme.spacing(3),
    marginBottom: theme.spacing(15),
  },
  largeMarginTop: {
    marginTop: theme.spacing(5),
  },
  marginTop: {
    marginTop: theme.spacing(3),
  },
  smallMarginTop: {
    marginTop: theme.spacing(1),
  },
  card: {
    border: "none",
    background: "none",
    width: "577px",
    [theme.breakpoints.down("sm")]: {
      width: "100%",
    },
  },
  captionText: {
    color: "#65727D",
    marginTop: theme.spacing(2),
    fontSize: "18px",
    lineHeight: "32.4px",
  },
  captionTextLargeMargin: {
    color: "#65727D",
    marginTop: theme.spacing(6),
    fontSize: "18px",
    lineHeight: "32.4px",
  },
  horizontalText: {
    marginTop: theme.spacing(5),
    marginBottom: theme.spacing(5),
    display: "flex",
  },
  textBox: {
    width: "157px",
    marginRight: theme.spacing(4),
  },
  divider: { width: "100%", marginTop: theme.spacing(7) },
}));

export interface OrderConfirmationStateProps {
  selectedOnlineService: OnlineService;
  selectedProduct: Product;
  selectedPack: Pack;
  orderNumber: string;
  onlineConsultation: OnlineQuestion[];
}

export interface OrderConfirmationDispatchProps {}

export type OrderConfirmationProps = OrderConfirmationStateProps &
  OrderConfirmationDispatchProps;

const OrderConfirmationPage: React.FC<OrderConfirmationProps> = ({
  selectedProduct,
  selectedPack,
  selectedOnlineService,
  orderNumber,
  onlineConsultation,
}) => {
  const classes = useStyles();
  useEffect(() => {
    analytics.trackEvent({
      flow: selectedOnlineService.id,
      event: "Order complete",
      metadata: {
        product: selectedProduct,
        packSize: selectedPack.name,
      },
    });
  }, [selectedOnlineService.id, selectedPack.name, selectedProduct]);
  return (
    <>
      <CheckoutHeader />
      <Container maxWidth={"sm"} className={classes.pageMargin}>
        <Card className={classes.card}>
          <Typography variant={"h2"} className={classes.largeMarginTop}>
            Thank you, we have received your order
          </Typography>
          <Typography variant={"h5"} className={classes.largeMarginTop}>
            Your order number {orderNumber}
          </Typography>
          <div className={classes.horizontalText}>
            <div className={classes.textBox}>
              <Typography variant={"h5"}>
                {`${selectedProduct.name} ${selectedProduct.variant ?? ""}`}
              </Typography>
            </div>
            <div className={classes.textBox}>
              <Typography variant={"h5"}>Estimated delivery</Typography>
              <Typography variant={"h5"}>2 to 4 days</Typography>
            </div>
          </div>
          <div className={classes.smallMarginTop}>
            <USPBanner product={selectedProduct} />
          </div>
          <Divider className={classes.divider} />
          <ImportantInfo consultation={onlineConsultation} />
          <Typography variant={"h2"} className={classes.largeMarginTop}>
            What happens next?
          </Typography>
          <Typography className={classes.captionTextLargeMargin}>
            Step 1
          </Typography>
          <Typography variant={"h3"} className={classes.smallMarginTop}>
            Pharmacist approval
          </Typography>
          <Typography className={classes.captionText}>
            One of our pharmacists will confirm you’re eligible for{" "}
            {selectedProduct.service === Ed.id ? "ED" : selectedProduct.name}{" "}
            treatment. This is to make sure your treatment is safe and suitable
            for you.
          </Typography>
          <Typography variant={"h5"} className={classes.marginTop}>
            What if my order isn't approved?
          </Typography>
          <Typography className={classes.captionText}>
            If our pharmacist can’t approve your order, they’ll explain the
            reason for this in an email.
          </Typography>
          <Typography className={classes.captionTextLargeMargin}>
            Step 2
          </Typography>
          <Typography variant={"h3"} className={classes.smallMarginTop}>
            Order dispatched
          </Typography>
          <Typography className={classes.captionText}>
            We’ll dispatch your order as soon as a pharmacist approves it.
            Dispatch will be on the same working day if you made your order
            before 3pm.
            <br />
            <br />
            If you ordered after 3pm, we’ll dispatch your order the next working
            day. If you ordered over the weekend, your order will be processed
            on the next working day.
          </Typography>
          <Typography className={classes.captionTextLargeMargin}>
            Step 3
          </Typography>
          <Typography variant={"h3"} className={classes.smallMarginTop}>
            Delivery
          </Typography>
          <Typography className={classes.captionText}>
            We’ll email you to let you know your order is out for delivery. It
            should be with you in 2-4 days.
          </Typography>
        </Card>
      </Container>
      <OnlineFooter />
    </>
  );
};

export default OrderConfirmationPage;
