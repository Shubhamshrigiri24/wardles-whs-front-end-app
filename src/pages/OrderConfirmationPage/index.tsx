import { connect } from "react-redux";
import { RootState } from "../../app/store/types";
import OrderConfirmationPage, {
  OrderConfirmationStateProps,
  OrderConfirmationDispatchProps,
} from "./OrderConfirmation";
import {
  getOnlineOrderNumber,
  getSelectedProduct,
  getSelectedOnlineService,
  getSelectedPack,
  getOnlineConsultation,
} from "../../app/store/selectors";

function mapStateToProps(state: RootState): OrderConfirmationStateProps {
  return {
    selectedOnlineService: getSelectedOnlineService(state),
    selectedProduct: getSelectedProduct(state),
    selectedPack: getSelectedPack(state),
    orderNumber: getOnlineOrderNumber(state),
    onlineConsultation: getOnlineConsultation(state),
  };
}

const mapDispatchToProps: OrderConfirmationDispatchProps = {};

export default connect(
  mapStateToProps,
  mapDispatchToProps
)(OrderConfirmationPage);
