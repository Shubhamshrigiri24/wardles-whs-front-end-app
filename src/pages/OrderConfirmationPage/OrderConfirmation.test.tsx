import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import OrderConfirmationPage from "./OrderConfirmation";
import { makePack, makeProduct } from "app/store/reducer/mock";
import { Router } from "react-router-dom";
import { createMemoryHistory } from "history";
import { Ed, Hana } from "app/store/reducer/online/types";

const history = createMemoryHistory();
const route = "/ed/checkout/confirmation";
history.push(route);

describe("<OrderConfirmationPage />", () => {
  it("should match the snapshot and have expected props for Ed", () => {
    const { container } = render(
      <Router history={history}>
        <OrderConfirmationPage
          selectedOnlineService={Ed}
          selectedProduct={makeProduct(undefined, {
            service: Ed.id,
            name: "viagra connect",
          })}
          selectedPack={makePack()}
          orderNumber="1234567"
          onlineConsultation={[]}
        />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });
  it("should match the snapshot and have expected props for Hana", () => {
    const { container } = render(
      <Router history={history}>
        <OrderConfirmationPage
          selectedOnlineService={Hana}
          selectedProduct={makeProduct(undefined, { service: Hana.id })}
          selectedPack={makePack()}
          orderNumber="1234567"
          onlineConsultation={[]}
        />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });
});
