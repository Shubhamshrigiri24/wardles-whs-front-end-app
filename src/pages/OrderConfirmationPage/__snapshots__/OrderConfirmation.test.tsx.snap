// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OrderConfirmationPage /> should match the snapshot and have expected props for Ed 1`] = `
<div>
  <div
    class="makeStyles-header-11"
  >
    <img
      alt="WellLogo"
      class="makeStyles-wellLogo-13"
      src="Well.svg"
    />
    <div
      class="makeStyles-textContainer-14"
    >
      <p
        class="MuiTypography-root MuiTypography-body1"
      >
        Pharmacy Services
      </p>
    </div>
  </div>
  <hr
    class="MuiDivider-root makeStyles-divider-12"
  />
  <div
    class="MuiContainer-root makeStyles-pageMargin-1 MuiContainer-maxWidthSm"
  >
    <div
      class="MuiPaper-root MuiCard-root makeStyles-card-5 MuiPaper-elevation1 MuiPaper-rounded"
    >
      <h2
        class="MuiTypography-root makeStyles-largeMarginTop-2 MuiTypography-h2"
      >
        Thank you, we have received your order
      </h2>
      <h5
        class="MuiTypography-root makeStyles-largeMarginTop-2 MuiTypography-h5"
      >
        Your order number 
        1234567
      </h5>
      <div
        class="makeStyles-horizontalText-8"
      >
        <div
          class="makeStyles-textBox-9"
        >
          <h5
            class="MuiTypography-root MuiTypography-h5"
          >
            viagra connect 
          </h5>
        </div>
        <div
          class="makeStyles-textBox-9"
        >
          <h5
            class="MuiTypography-root MuiTypography-h5"
          >
            Estimated delivery
          </h5>
          <h5
            class="MuiTypography-root MuiTypography-h5"
          >
            2 to 4 days
          </h5>
        </div>
      </div>
      <div
        class="makeStyles-smallMarginTop-4"
      >
        <div
          class="makeStyles-root-15 makeStyles-infoType-16"
        >
          <div
            class="makeStyles-icon-17"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M20 8h-3V4H3c-1.1 0-2 .9-2 2v11h2c0 1.66 1.34 3 3 3s3-1.34 3-3h6c0 1.66 1.34 3 3 3s3-1.34 3-3h2v-5l-3-4zM6 18.5c-.83 0-1.5-.67-1.5-1.5s.67-1.5 1.5-1.5 1.5.67 1.5 1.5-.67 1.5-1.5 1.5zm13.5-9l1.96 2.5H17V9.5h2.5zm-1.5 9c-.83 0-1.5-.67-1.5-1.5s.67-1.5 1.5-1.5 1.5.67 1.5 1.5-.67 1.5-1.5 1.5z"
              />
            </svg>
          </div>
          <div
            class="makeStyles-message-18"
          >
            This item has free delivery. It is delivered in discreet unbranded packaging.
          </div>
        </div>
      </div>
      <hr
        class="MuiDivider-root makeStyles-divider-10"
      />
      <h2
        class="MuiTypography-root makeStyles-header-19 MuiTypography-h2"
      >
        Important information about your order
      </h2>
      <h2
        class="MuiTypography-root makeStyles-largeMarginTop-2 MuiTypography-h2"
      >
        What happens next?
      </h2>
      <p
        class="MuiTypography-root makeStyles-captionTextLargeMargin-7 MuiTypography-body1"
      >
        Step 1
      </p>
      <h3
        class="MuiTypography-root makeStyles-smallMarginTop-4 MuiTypography-h3"
      >
        Pharmacist approval
      </h3>
      <p
        class="MuiTypography-root makeStyles-captionText-6 MuiTypography-body1"
      >
        One of our pharmacists will confirm you’re eligible for
         
        ED
         
        treatment. This is to make sure your treatment is safe and suitable for you.
      </p>
      <h5
        class="MuiTypography-root makeStyles-marginTop-3 MuiTypography-h5"
      >
        What if my order isn't approved?
      </h5>
      <p
        class="MuiTypography-root makeStyles-captionText-6 MuiTypography-body1"
      >
        If our pharmacist can’t approve your order, they’ll explain the reason for this in an email.
      </p>
      <p
        class="MuiTypography-root makeStyles-captionTextLargeMargin-7 MuiTypography-body1"
      >
        Step 2
      </p>
      <h3
        class="MuiTypography-root makeStyles-smallMarginTop-4 MuiTypography-h3"
      >
        Order dispatched
      </h3>
      <p
        class="MuiTypography-root makeStyles-captionText-6 MuiTypography-body1"
      >
        We’ll dispatch your order as soon as a pharmacist approves it. Dispatch will be on the same working day if you made your order before 3pm.
        <br />
        <br />
        If you ordered after 3pm, we’ll dispatch your order the next working day. If you ordered over the weekend, your order will be processed on the next working day.
      </p>
      <p
        class="MuiTypography-root makeStyles-captionTextLargeMargin-7 MuiTypography-body1"
      >
        Step 3
      </p>
      <h3
        class="MuiTypography-root makeStyles-smallMarginTop-4 MuiTypography-h3"
      >
        Delivery
      </h3>
      <p
        class="MuiTypography-root makeStyles-captionText-6 MuiTypography-body1"
      >
        We’ll email you to let you know your order is out for delivery. It should be with you in 2-4 days.
      </p>
    </div>
  </div>
  <div
    class="makeStyles-container-23"
  >
    <hr
      class="MuiDivider-root"
    />
    <div
      class="makeStyles-upperSection-25"
    >
      <div
        class="makeStyles-upperCaptionContainer-27"
      >
        <a
          href="https://www.well.co.uk/"
        >
          <img
            alt="wellLogo"
            class="makeStyles-wellLogo-36"
            src="WellLogo.svg"
          />
        </a>
        <p
          class="MuiTypography-root makeStyles-caption-24 MuiTypography-body1"
        >
          This content was written and edited by Well. Our health information is evidence based, up-to-date and reviewed by health professionals. It isn’t intended to replace the medical information given to you by your doctor. We aim to provide you with the knowledge and support you need to make confident decisions about your health and wellbeing.
        </p>
      </div>
      <div
        class="makeStyles-linksList-29"
      >
        <h6
          class="MuiTypography-root makeStyles-links-30 MuiTypography-h6"
        >
          <a
            class="makeStyles-linkStyle-37"
            href="https://www.well.co.uk/about-us/policies/terms-and-conditions-vaccination-services"
          >
            Terms and conditions
          </a>
        </h6>
        <h6
          class="MuiTypography-root makeStyles-links-30 MuiTypography-h6"
        >
          <a
            class="makeStyles-linkStyle-37"
            href="https://www.well.co.uk/about-us/policies/privacy"
          >
            Privacy policy
          </a>
        </h6>
        <h6
          class="MuiTypography-root makeStyles-links-30 MuiTypography-h6"
        >
          <a
            class="makeStyles-linkStyle-37"
            href="https://www.well.co.uk/about-us/policies/cookies"
          >
            Cookies
          </a>
        </h6>
      </div>
      <div
        class="makeStyles-supportEmail-31"
      >
        <p
          class="MuiTypography-root MuiTypography-body1"
        >
          For support email
        </p>
        <h6
          class="MuiTypography-root MuiTypography-h6"
        >
          <a
            class="makeStyles-linkStyle-37"
            href="mailto:hello@well.co.uk"
          >
            hello@well.co.uk
          </a>
        </h6>
      </div>
    </div>
    <hr
      class="MuiDivider-root"
    />
    <div
      class="makeStyles-lowerSection-26"
    >
      <div
        class="makeStyles-lowerCaptionContainer-28"
      >
        <p
          class="MuiTypography-root makeStyles-caption-24 MuiTypography-body1"
        >
          Bestway National Chemists Limited is registered in England and Wales, trading as Well and Well Pharmacy. Our online pharmacy (well.co.uk) registration number is 9010492 and the registered pharmacy address is: Well, Healthcare Service Centre, Meir Park, Stoke-on-Trent, Staffordshire, ST3 7UN. If you would like to know who the Responsible Pharmacist is at any given time, please email nhspharmacy.stoke.well.co.ukfap20@nhs.net or call 01782 597313. You can also contact the pharmacy on hello@well.co.uk. Our Superintendent Pharmacist is Iftkhar Ahmad Khan, FRPharmS. GPhC Registration Number: 2041286. You can find out more about checking if an online pharmacy is operating legally from the General Pharmaceutical Council (GPhC).
        </p>
      </div>
      <div
        class="makeStyles-logoContainer-35"
      >
        <a
          href="https://www.nhs.uk/"
        >
          <img
            alt="nhs"
            class="makeStyles-logo1-32"
            src="NHS.png"
          />
        </a>
        <a
          href="https://www.legitscript.com/websites/well.co.uk"
        >
          <img
            alt="well"
            class="makeStyles-logo2-33"
            src="Well.svg"
          />
        </a>
      </div>
    </div>
  </div>
</div>
`;

exports[`<OrderConfirmationPage /> should match the snapshot and have expected props for Hana 1`] = `
<div>
  <div
    class="makeStyles-header-48"
  >
    <img
      alt="WellLogo"
      class="makeStyles-wellLogo-50"
      src="Well.svg"
    />
    <div
      class="makeStyles-textContainer-51"
    >
      <p
        class="MuiTypography-root MuiTypography-body1"
      >
        Pharmacy Services
      </p>
    </div>
  </div>
  <hr
    class="MuiDivider-root makeStyles-divider-49"
  />
  <div
    class="MuiContainer-root makeStyles-pageMargin-38 MuiContainer-maxWidthSm"
  >
    <div
      class="MuiPaper-root MuiCard-root makeStyles-card-42 MuiPaper-elevation1 MuiPaper-rounded"
    >
      <h2
        class="MuiTypography-root makeStyles-largeMarginTop-39 MuiTypography-h2"
      >
        Thank you, we have received your order
      </h2>
      <h5
        class="MuiTypography-root makeStyles-largeMarginTop-39 MuiTypography-h5"
      >
        Your order number 
        1234567
      </h5>
      <div
        class="makeStyles-horizontalText-45"
      >
        <div
          class="makeStyles-textBox-46"
        >
          <h5
            class="MuiTypography-root MuiTypography-h5"
          >
            name-default 
          </h5>
        </div>
        <div
          class="makeStyles-textBox-46"
        >
          <h5
            class="MuiTypography-root MuiTypography-h5"
          >
            Estimated delivery
          </h5>
          <h5
            class="MuiTypography-root MuiTypography-h5"
          >
            2 to 4 days
          </h5>
        </div>
      </div>
      <div
        class="makeStyles-smallMarginTop-41"
      >
        <div
          class="makeStyles-root-52 makeStyles-infoType-53"
        >
          <div
            class="makeStyles-icon-54"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 21.35l-1.45-1.32C5.4 15.36 2 12.28 2 8.5 2 5.42 4.42 3 7.5 3c1.74 0 3.41.81 4.5 2.09C13.09 3.81 14.76 3 16.5 3 19.58 3 22 5.42 22 8.5c0 3.78-3.4 6.86-8.55 11.54L12 21.35z"
              />
            </svg>
          </div>
          <div
            class="makeStyles-message-55"
          >
            Includes ongoing support from our pharmacists.
          </div>
        </div>
      </div>
      <hr
        class="MuiDivider-root makeStyles-divider-47"
      />
      <h2
        class="MuiTypography-root makeStyles-header-56 MuiTypography-h2"
      >
        Important information about your order
      </h2>
      <h2
        class="MuiTypography-root makeStyles-largeMarginTop-39 MuiTypography-h2"
      >
        What happens next?
      </h2>
      <p
        class="MuiTypography-root makeStyles-captionTextLargeMargin-44 MuiTypography-body1"
      >
        Step 1
      </p>
      <h3
        class="MuiTypography-root makeStyles-smallMarginTop-41 MuiTypography-h3"
      >
        Pharmacist approval
      </h3>
      <p
        class="MuiTypography-root makeStyles-captionText-43 MuiTypography-body1"
      >
        One of our pharmacists will confirm you’re eligible for
         
        name-default
         
        treatment. This is to make sure your treatment is safe and suitable for you.
      </p>
      <h5
        class="MuiTypography-root makeStyles-marginTop-40 MuiTypography-h5"
      >
        What if my order isn't approved?
      </h5>
      <p
        class="MuiTypography-root makeStyles-captionText-43 MuiTypography-body1"
      >
        If our pharmacist can’t approve your order, they’ll explain the reason for this in an email.
      </p>
      <p
        class="MuiTypography-root makeStyles-captionTextLargeMargin-44 MuiTypography-body1"
      >
        Step 2
      </p>
      <h3
        class="MuiTypography-root makeStyles-smallMarginTop-41 MuiTypography-h3"
      >
        Order dispatched
      </h3>
      <p
        class="MuiTypography-root makeStyles-captionText-43 MuiTypography-body1"
      >
        We’ll dispatch your order as soon as a pharmacist approves it. Dispatch will be on the same working day if you made your order before 3pm.
        <br />
        <br />
        If you ordered after 3pm, we’ll dispatch your order the next working day. If you ordered over the weekend, your order will be processed on the next working day.
      </p>
      <p
        class="MuiTypography-root makeStyles-captionTextLargeMargin-44 MuiTypography-body1"
      >
        Step 3
      </p>
      <h3
        class="MuiTypography-root makeStyles-smallMarginTop-41 MuiTypography-h3"
      >
        Delivery
      </h3>
      <p
        class="MuiTypography-root makeStyles-captionText-43 MuiTypography-body1"
      >
        We’ll email you to let you know your order is out for delivery. It should be with you in 2-4 days.
      </p>
    </div>
  </div>
  <div
    class="makeStyles-container-60"
  >
    <hr
      class="MuiDivider-root"
    />
    <div
      class="makeStyles-upperSection-62"
    >
      <div
        class="makeStyles-upperCaptionContainer-64"
      >
        <a
          href="https://www.well.co.uk/"
        >
          <img
            alt="wellLogo"
            class="makeStyles-wellLogo-73"
            src="WellLogo.svg"
          />
        </a>
        <p
          class="MuiTypography-root makeStyles-caption-61 MuiTypography-body1"
        >
          This content was written and edited by Well. Our health information is evidence based, up-to-date and reviewed by health professionals. It isn’t intended to replace the medical information given to you by your doctor. We aim to provide you with the knowledge and support you need to make confident decisions about your health and wellbeing.
        </p>
      </div>
      <div
        class="makeStyles-linksList-66"
      >
        <h6
          class="MuiTypography-root makeStyles-links-67 MuiTypography-h6"
        >
          <a
            class="makeStyles-linkStyle-74"
            href="https://www.well.co.uk/about-us/policies/terms-and-conditions-vaccination-services"
          >
            Terms and conditions
          </a>
        </h6>
        <h6
          class="MuiTypography-root makeStyles-links-67 MuiTypography-h6"
        >
          <a
            class="makeStyles-linkStyle-74"
            href="https://www.well.co.uk/about-us/policies/privacy"
          >
            Privacy policy
          </a>
        </h6>
        <h6
          class="MuiTypography-root makeStyles-links-67 MuiTypography-h6"
        >
          <a
            class="makeStyles-linkStyle-74"
            href="https://www.well.co.uk/about-us/policies/cookies"
          >
            Cookies
          </a>
        </h6>
      </div>
      <div
        class="makeStyles-supportEmail-68"
      >
        <p
          class="MuiTypography-root MuiTypography-body1"
        >
          For support email
        </p>
        <h6
          class="MuiTypography-root MuiTypography-h6"
        >
          <a
            class="makeStyles-linkStyle-74"
            href="mailto:hello@well.co.uk"
          >
            hello@well.co.uk
          </a>
        </h6>
      </div>
    </div>
    <hr
      class="MuiDivider-root"
    />
    <div
      class="makeStyles-lowerSection-63"
    >
      <div
        class="makeStyles-lowerCaptionContainer-65"
      >
        <p
          class="MuiTypography-root makeStyles-caption-61 MuiTypography-body1"
        >
          Bestway National Chemists Limited is registered in England and Wales, trading as Well and Well Pharmacy. Our online pharmacy (well.co.uk) registration number is 9010492 and the registered pharmacy address is: Well, Healthcare Service Centre, Meir Park, Stoke-on-Trent, Staffordshire, ST3 7UN. If you would like to know who the Responsible Pharmacist is at any given time, please email nhspharmacy.stoke.well.co.ukfap20@nhs.net or call 01782 597313. You can also contact the pharmacy on hello@well.co.uk. Our Superintendent Pharmacist is Iftkhar Ahmad Khan, FRPharmS. GPhC Registration Number: 2041286. You can find out more about checking if an online pharmacy is operating legally from the General Pharmaceutical Council (GPhC).
        </p>
      </div>
      <div
        class="makeStyles-logoContainer-72"
      >
        <a
          href="https://www.nhs.uk/"
        >
          <img
            alt="nhs"
            class="makeStyles-logo1-69"
            src="NHS.png"
          />
        </a>
        <a
          href="https://www.legitscript.com/websites/well.co.uk"
        >
          <img
            alt="well"
            class="makeStyles-logo2-70"
            src="Well.svg"
          />
        </a>
      </div>
    </div>
  </div>
</div>
`;
